crates/cast/src/lib.rs

```diff
@@ -187,7 +183,13 @@ fn get_account_from_keystore<'a>(
             .ok_or_else(|| anyhow::anyhow!("Failed to get address from account JSON file - make sure the account is deployed"))?
     )?;
 
-    Ok(SingleOwnerAccount::new(provider, signer, address, chain_id))
+    Ok(SingleOwnerAccount::new(
+        provider,
+        signer,
+        address,
+        chain_id,
+        ExecutionEncoding::Legacy,
```

Comment:
Why `::Legacy`?

Reply:
We'll do migration to cairo1 accounts in separate task (as this has to be adjusted in devnet as well for proper testing)